sdk/test-utils/test-credential/CHANGELOG.md

```diff
@@ -4,11 +4,11 @@
 
 ### Breaking Changes
 
-Updates the `createTestCredential` method to consume `ChainedTokenCredential` instead of `ClientSecretCredential` in order to offer autonomy to the devs and to move away from client secrets in environment varaibles.
+Updates the `createTestCredential` method to consume `AzurePipelineCredential` when it is running in Azure Pipeline environment or `ChainedTokenCredential` otherwise instead of `ClientSecretCredential`. This offers autonomy to the devs, increases security running in Dev Ops environment, and moves away from client secrets in environment variables.
 
 - `NoOpCredential` is offered for playback.
 - In record and live modes:
-  - `ChainedTokenCredential` is offered in Node. The credential will try `AzurePowershelLCredential`, `AzureCliCredential`, `EnvironmentCredential`, and `AzureDeveloperCliCredential` in the listed order.
+  - In Node, `AzurePipelinesCredential` is run when the environment is Azure Pipelines with service connection. Otherwise `ChainedTokenCredential` is offered. The credential will try `AzurePowershelLCredential`, `AzureCliCredential`, `EnvironmentCredential`, and `AzureDeveloperCliCredential` in the listed order.
   - In the browser, a custom credential is provided that fetches tokens from a locally running Node server. The server is provided in the dev-tool package, and must be running while the browser
     tests are running for the credential to work. The server uses `ChainedTokenCredential` on the host machine to generate tokens.
```

Comment:
I think we might need to update the browser relay in dev-tool as well

Reply:
I tried but sadly Azure Pipelines Credential does not work in the browser :(

Reply:
See [this pipeline run](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4025964&view=logs&j=8f098e13-557e-5a76-9331-06424800e0fa&t=52fc44a1-b208-5374-c579-e601a962726b&l=137)

Reply:
Ohh thank you for the clarification. Makes sense and updating it now!